VERSION

```diff
@@ -0,0 +1 @@
+v1beta2-1.6.2-3.5.0
```

Comment:
@ChenYi015 Improving the release process is out of scope of this PR: https://github.com/kubeflow/spark-operator/issues/1975 ?
E.g. that we should keep the same version for Spark Operator image and Helm charts release.

Reply:
> @ChenYi015 Improving the release process is out of scope of this PR: #1975 ? E.g. that we should keep the same version for Spark Operator image and Helm charts release.

@andreyvelich I have updated the doc to keep the versions the same when creating a git tag.

```
VERSION=$(cat VERSION)
sed -i "s/^version.*/version: ${VERSION}/" charts/spark-operator-chart/Chart.yaml
sed -i "s/^appVersion.*/appVersion: ${VERSION}/" charts/spark-operator-chart/Chart.yaml
```

Reply:
@ChenYi015 Since we keep Chart and Image version the same, why do we need `appVersion` parameter in Helm Charts: https://github.com/kubeflow/spark-operator/blob/a8ee09a3a6a4cf3140c0ea8b2c3eac7b057c1090/charts/spark-operator-chart/Chart.yaml#L5

For example, KServe has only `version` [in the Chart file ](https://github.com/kserve/kserve/blob/master/charts/kserve-resources/Chart.yaml#L3)

Reply:
@andreyvelich `appVersion` is indeed an optional field, but it can give information about the operator version, when helm users search charts like this, the `APP VERSION` will be shown:

```bash
$ helm search repo spark-operator/spark-operator                                         
NAME                            CHART VERSION   APP VERSION             DESCRIPTION                                   
spark-operator/spark-operator   2.0.0           2.0.0                   A Helm chart for Spark on Kubernetes operator
``` 

And if we removed it, it will be like this and may cause some confusion:

```bash
$ helm search repo spark-operator/spark-operator                                         
NAME                            CHART VERSION   APP VERSION             DESCRIPTION                                   
spark-operator/spark-operator   2.0.0                                   A Helm chart for Spark on Kubernetes operator
```

Besides, we will not need to modify the chart templates if we choose to keep the `appVersion` field, as all resources created by the chart will have `appVersion` labels.

Reply:
Do you think it is necessary for users since Chart version is always equal to the Image version ?

Reply:
Though they are the same, I think it is better to keep the `appVersion` field as it provides extra metadata information.